### PR TITLE
Weather#10

### PR DESCRIPTION
**Pull request by venkykatta**
Display all weather information

### CODE REVIEW

**Comment by su66uu** [comment]
Can you give a proper title for the PR?

File: app.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-from flask import Flask, request, render_template , url_for , redirect  
+from flask import Flask, request, render_template, url_for , redirect  
 import requests  
 
 from markupsafe import escape
```
Comment (su66uu) [inline app.py]: It looks you did not use this anywhere?

**Review by su66uu — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (venkykatta) [reply app.py]: I resolve it

File: app.py [hunk 2]
```diff
@@ -18,14 +18,20 @@ def Onsubmit():
...
+        'humidity' : weather_info["main"]["humidity"],
+        'wind_speed' : weather_info["wind"]["speed"],
+        'clouds' : weather_info["clouds"]["all"],
+        'country' : weather_info["sys"]["country"]
     }
     
-    
-    return render_template('bulmacard.html', weather=weather)
-
+    return render_template('bulmacard.html', weather = weather)
```
Comment (su66uu) [inline app.py]: If you look at the above two lines, there is no logic change. It is just a spaces difference. So the PR should not contain these type of changes

**Review by su66uu — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (su66uu) [reply app.py]: I still see the escape import. Can you check it?

Comment (venkykatta) [reply app.py]: Sorry, I resolve it.

File: app.py (hunk 2 above)
Comment (venkykatta) [reply app.py]: Yeah, there is no logic those lines. i definitely change that.

Comment (su66uu) [reply app.py]: This is happening often. Can you find a solution for this?

**Review by su66uu — APPROVED** [review]

File: app.py (hunk 2 above)
Comment (venkykatta) [reply app.py]: Yeah.. that was correct subbu.

**Pull request merged**